assets/src/edit-story/components/inspector/documentInspector.js

```diff
@@ -1,9 +1,187 @@
+	font-size: 14px;
+	width: 100%;
+	padding: 15px;
+	background: none;
+	margin: 5px 0px;
+`;
+const Img = styled.img`
+	width: 100%;
+	max-height: 300px
+`;
+
+const Group = styled.div`
+	border-color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	display: block;
+	align-items: center;
+	margin: 15px 0px;
+`;
+
+const RemoveButton = styled.button`
+	${ ButtonCSS }
+`;
 
 function DocumentInspector() {
+	const {
+		actions: { loadStatuses, loadUsers, setDisabledStatuses },
+		state: { users, statuses, disabledStatuses },
+	} = useInspector();
+
+	const {
+		state: { meta: { isSaving }, story: { author, status, slug, date, excerpt, featuredMediaUrl, password }, capabilities },
+		actions: { updateStory, deleteStory },
+	} = useStory();
+
+	const { postThumbnails } = useConfig();
+
+	useEffect( () => {
+		loadStatuses();
+		loadUsers();
+	} );
+
+	useEffect( () => {
+		const states = ( status === 'future' ) ? [ 'pending' ] : [ 'future', 'pending' ];
+		setDisabledStatuses( states );
+	}, [ status, setDisabledStatuses ] );
+
+	const handleChangeStatus = useCallback(
+		( evt ) => 	updateStory( { properties: { status: evt.target.value } } ),
+		[ updateStory ],
+	);
+
+	const handleChangeAuthor = useCallback(
+		( evt ) => updateStory( { properties: { author: evt.target.value } } ),
+		[ updateStory ],
+	);
+
+	const handleChangePassword = useCallback(
+		( value, evt ) => updateStory( { properties: { password: evt.target.value } } ),
+		[ updateStory ],
+	);
+
+	const handleChangeDate = useCallback(
+		( value, evt ) => updateStory( { properties: { date: evt.target.value } } ),
+		[ updateStory ],
+	);
+
+	const handleChangeExcerpt = useCallback(
+		( value, evt ) => updateStory( { properties: { excerpt: evt.target.value } } ),
+		[ updateStory ],
+	);
+
+	const handleChangeSlug = useCallback(
+		( value, evt ) => updateStory( { properties: { slug: evt.target.value } } ),
+		[ updateStory ],
+	);
+
+	const handleChangeImage = useCallback(
+		( image ) => updateStory( { properties: { featuredMedia: image.id, featuredMediaUrl: ( image.sizes && image.sizes.medium ) ? image.sizes.medium.url : image.url } } ),
+		[ updateStory ],
+	);
+
+	const handleRemoveImage = useCallback(
+		( evt ) => {
+			updateStory( { properties: { featuredMedia: 0, featuredMediaUrl: '' } } );
+			evt.preventDefault();
+		},	[ updateStory ],
+	);
+
+	const handleRemoveStory = useCallback(
+		( evt ) => {
+			deleteStory();
+			evt.preventDefault();
+		},	[ deleteStory ],
+	);
+
 	return (
-		<h2>
-			{ 'Document' }
-		</h2>
+		<>
+			<h2>
+				{ 'Document' }
+			</h2>
+			{ capabilities && capabilities.hasPublishAction && statuses && <SelectMenu
+				label="Status"
+				name="status"
+				options={ statuses }
+				disabled={ isSaving }
+				disabledOptions={ disabledStatuses }
+				value={ status }
+				onChange={ handleChangeStatus }
+			/> }
+			{ capabilities && capabilities.hasPublishAction && status !== 'private' && <InputGroup
```

Comment:
When testing, I currently can't see the option for setting the visibility to Password Protected (just the password field). Not sure if I'm missing something?

Reply:
You are not. Current there is not check to make password protected. Currently, a post becomes password protected by adding a password. I will change this when UX comes in.